vaadin-components-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridTHTDElement.java

```diff
@@ -27,8 +29,11 @@
 
     @Override
     public String getText() {
-        TestBenchElement slot = getPropertyElement("firstElementChild");
-        return getSlotText(slot);
+        return (String) executeScript("var cell = arguments[0];"
```

Comment:
This might warrant an explaining comment

Reply:
In #14